static/app/components/u2f/u2finterface.tsx

```diff
@@ -56,13 +60,27 @@ class U2fInterface extends React.Component<Props, State> {
 
   getU2FResponse(data) {
     if (data.response) {
-      const authenticatorData = {
-        keyHandle: data.id,
-        clientData: bufferToBase64url(data.response.clientDataJSON),
-        signatureData: bufferToBase64url(data.response.signature),
-        authenticatorData: bufferToBase64url(data.response.authenticatorData),
-      };
-      return JSON.stringify(authenticatorData);
+      if (this.props.flowMode === 'sign') {
```

Comment:
is this backwards compatible with current u2f? why didn't we previously need to check the flowMode here?

Reply:
yup, its used below in the old u2f flow in `invokeU2fFlow` we didn't need to check previously in `getU2FResponse` as we didn't need to generate a new object for the registration/verification data